Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org

```diff
@@ -1548,6 +1608,19 @@ System.Nullable<
             {
             }
         }
+        public static void getCustomerPaymentProfileNonceRequest(getCustomerPaymentProfileNonceRequest argument) 
+        {
+            if(null != argument) 
+            {
+            }
+        }
```

Comment:
This is redundant. Add some code here or remove the if block.

Reply:
Removed the IF block.